tailscale/client.go

```diff
@@ -518,6 +517,7 @@ type (
 		Created      time.Time       `json:"created"`
 		Expires      time.Time       `json:"expires"`
 		Capabilities KeyCapabilities `json:"capabilities"`
+		Expiry       int             `json:"expirySeconds"`
```

Comment:
This won't work. This is the response struct, not the request struct.
See line 530 on how the request is being constructed.

You'll need to replace
```
map[string]KeyCapabilities{
	"capabilities": capabilities,
}
```
with
```
map[string]any{
	"capabilities": capabilities,
	"expirySeconds": expirySeconds
}
```

Which also means that you will need to add a parameter to `CreateKey` but that breaks backwards compat. So you could instead add `CreateKeyWithExpiry(ctx context.Context, capabilities KeyCapabilities, expirySeconds int)` or `CreateKeyWithOpts(ctx context.Context, capabilities KeyCapabilities, opts ...CreateKeyOpt)`